tests/test_histogram_indexing.py

```diff
@@ -205,6 +205,29 @@ def test_mix_value_with_slice_2():
     assert_array_equal(h2.shape, (5, 2, 2))
 
 
+def test_one_sided_slice():
+    h = bh.Histogram(bh.axis.Regular(4, 1, 5))
+    h.view(True)[:] = 1
+
+    assert h[sum] == 6  # 4 (internal bins) + 2 (flow bins)
+    assert h[bh.tag.at(-1) : bh.tag.at(5) : sum] == 6  # keeps underflow, keeps overflow
```

Comment:
UHI follows normal Python indexing, so since `h[-1:5:sum]` should match `h[-1:5]`, which is a valid NumPy expression, the original form of this line was incorrect. I've corrected it to use `bh.tag.at()`, which is boost-histogram style -1 to N+1 indexing.